app/src/main/java/com/blink22/android/mvpandroid/di/NetworkModule.java

```diff
@@ -0,0 +1,91 @@
+package com.blink22.android.mvpandroid.di;
+
+import android.app.Application;
+
+import com.blink22.android.mvpandroid.BuildConfig;
+import com.blink22.android.mvpandroid.apiinterfaces.TodosService;
+import com.blink22.android.mvpandroid.network.TodosSubscriber;
+import com.google.gson.Gson;
+
+import java.io.File;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import dagger.Module;
+import dagger.Provides;
+import okhttp3.Cache;
+import okhttp3.OkHttpClient;
+import retrofit2.CallAdapter;
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Created by ahmedghazy on 7/29/18.
+ */
+
+@Module
+public class NetworkModule {
+    @Provides
+    @Named("api_url")
```

Comment:
@AFGhazy I believe ` @Named("api_url")` is redundant and can be removed. What do you think?